### ISSUE TITLE
RGBA Images training

### ISSUE THREAD

**Issue by vijishmadhavan** [eps696/stylegan2ada#24]
I am trying to train with rgba images, but its going to gradient explosion, I tried to reduce the learning rate but nothing seems to work.
Can somebody help? My dataset has about 2k unique images and I pumped it upto 60k images with augmentation.
Finetuning by adding alpha to other rgb models did not go well, Im training from scratch.
![download (31)](https://user-images.githubusercontent.com/53169213/202849400-7750b86e-f416-4b43-a0a9-c7ea990b0cd6.png)

**Comment by eps696** [comment]
you may try disabling `apa` augmentation (on by default) by adding `--aug ada`. in my tests `apa` enhanced generation variety, but i didn't try it with alpha yet.
